lib/ansible/modules/windows/win_psexec.ps1

```diff
@@ -0,0 +1,189 @@
+#!powershell
+# This file is part of Ansible
+#
+# Copyright 2017, Dag Wieers (@dagwieers) <dag@wieers.com>
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+# WANT_JSON
+# POWERSHELL_COMMON
+
+# See also: https://technet.microsoft.com/en-us/sysinternals/pxexec.aspx
+
+$params = Parse-Args $args
+
+$command = Get-AnsibleParam -obj $params -name "command" -failifempty $true
+$executable = Get-AnsibleParam -obj $params -name "executable" -default "psexec.exe"
+$hostnames = Get-AnsibleParam -obj $params -name "hostnames"
+$username = Get-AnsibleParam -obj $params -name "username"
+$password = Get-AnsibleParam -obj $params -name "password"
+$chdir = Get-AnsibleParam -obj $params -name "chdir" -type "path"
+$wait = Get-AnsibleParam -obj $params -name "wait" -type "bool" -default $true
+$noprofile = Get-AnsibleParam -obj $params -name "noprofile" -type "bool" -default $false
+$elevated = Get-AnsibleParam -obj $params -name "elevated" -type "bool" -default $false
+$limited = Get-AnsibleParam -obj $params -name "limited" -type "bool" -default $false
+$system = Get-AnsibleParam -obj $params -name "system" -type "bool" -default $false
+$interactive = Get-AnsibleParam -obj $params -name "interactive" -type "bool" -default $false
+$priority = Get-AnsibleParam -obj $params -name "priority" -validateset "background","low","belownormal","abovenormal","high","realtime"
+$timeout = Get-AnsibleParam -obj $params -name "timeout"
+$extra_opts = Get-AnsibleParam -obj $params -name "extra_opts" -default @()
+
+$result = @{
+    changed = $true
+}
+
+If (-Not (Get-Command $executable)) {
```

Comment:
I think you need -ErrorAction SilentlyContinue on this one or it bombs under Strict + Stop.